### PR TITLE
Retire the `mutect-vs-strelka` analysis

### PR DESCRIPTION
**Pull request by cansavvy**
#### Purpose/implementation

The `mutect-vs-strelka` analysis was the original attempt to find consensus mutation calls between the two callers. The results of this analysis led to adding [VarDict and Lancet as a callers](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/103). Since the addition of these callers, the snv caller analysis had too be overhauled to accommodate and compare 4 callers. 

This means `mutect-vs-strelka` is out of date but should still be kept around for record keeping purposes. To this end, I've added a README to explain the evolution of this analysis and have also removed it from CircleCI tests.  

A minor but related change in this PR, I changed the plots from being saved to a PDF to a png so they could be added to the README. 

#### Issue

This pertains to #30 and #103 which have been superseded by the issue #161 

#### Directions for reviewers

Does the README make sense? Is there any information that should be added that I am missing? 

#### Results

NA. There are no new results, I am just retiring old ones. 

#### Docker and continuous integration

- [x] This analysis has been removed from continuous integration.

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
Can you tell me about the `combined_results.tsv.zip` addition? I didn't spot any changes in the code such that I would expect to see this file added.

File: analyses/mutect2-vs-strelka2/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+to break these variants into their single nucleotide changes.
+In these analyses, these base changes have been grouped together and collectively
+called `long_changes`.
+The higher base resolution of Strelka2, and its ability to parse apart the SNVs
+from each other, is more useful to us for this particular analyses, as the larger
+structural variants are better detected in the Manta or LUMPY analyses.
+
+### Gene Summary
+
+![](/plots/gene_summary_cor_strelka-vs-mutect2.png)
```
Comment (jaclyn-taroni) [inline analyses/mutect2-vs-strelka2/README.md]: I think you may want 
```suggestion
![](plots/gene_summary_cor_strelka-vs-mutect2.png)
```
It's not showing up for me in the rich diff.

Comment (cansavvy) [reply analyses/mutect2-vs-strelka2/README.md]: Strange. My Atom "GitHub Markdown" preview shows it correctly.

Comment (cgreene) [reply analyses/mutect2-vs-strelka2/README.md]: The leading `/` makes it an absolute path, so you'll definitely want to remove that as @jaclyn-taroni suggests. Is this in the root directory of some Atom project, so Atom interprets `/` as equivalent to a relative path? I don't know why it'd be set that way, but it's the only thing that comes to mind.

**Comment by cansavvy** [comment]
> Can you tell me about the `combined_results.tsv.zip` addition? I didn't spot any changes in the code such that I would expect to see this file added.

The code for this file creation was always there, I just hadn't uploaded it to GitHub previously. I am unsure why. It could be the past version of the file were too large for GitHub.

**Comment by jaclyn-taroni** [comment]
> The code for this file creation was always there, I just hadn't uploaded it to GitHub previously. I am unsure why. It could be the past version of the file were too large for GitHub.

Okay, sounds good!

File: analyses/mutect2-vs-strelka2/README.md (hunk 1 above)
Comment (cansavvy) [reply analyses/mutect2-vs-strelka2/README.md]: Fixed it.

**Pull request merged**